### PR TITLE
Add ensemble CI test for Python backend

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

File: qa/python_models/add_sub/config.pbtxt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+]
+output [
+  {
+    name: "OUTPUT1"
+    data_type: TYPE_FP32
+    dims: [ 4 ]
+  }
+]
+
+instance_group [{ kind: KIND_CPU }]
```
Comment (CoderHam) [inline qa/python_models/add_sub/config.pbtxt:59]: Add newline

File: qa/python_models/ensemble/config.pbtxt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        key: "OUTPUT0"
+        value: "OUTPUT0"
+      }
+      output_map {
+        key: "OUTPUT1"
+        value: "OUTPUT1"
+      }
+    }
+  ]
+}
```
Comment (CoderHam) [inline qa/python_models/ensemble/config.pbtxt:72]: Add newline

**Review by CoderHam — CHANGES_REQUESTED** [review]
You can remove 
```
import tritonclient.grpc as grpcclient
```

from `python_test.py` and also use httpclient directly instead of client_util since you are only running the test with the HTTP client library.

File: qa/python_models/add_sub/config.pbtxt (hunk 1 above)
Comment (Tabrizian) [reply qa/python_models/add_sub/config.pbtxt:59]: It has a new line. It just doesn't show it.

File: qa/L0_backend_python/python_test.py [hunk 3]
```diff
@@ -208,6 +201,29 @@ def test_init_args(self):
...
+                                      np_to_triton_dtype(input_data_0.dtype)),
+                httpclient.InferInput("INPUT1", input_data_1.shape,
+                                      np_to_triton_dtype(input_data_1.dtype))
+            ]
+            inputs[0].set_data_from_numpy(input_data_0)
+            inputs[1].set_data_from_numpy(input_data_1)
+            result = client.infer(model_name, inputs)
+            output0 = result.as_numpy('OUTPUT0')
+            output1 = result.as_numpy('OUTPUT1')
+            self.assertIsNotNone(output0)
```
Comment (deadeyegoodwin) [inline qa/L0_backend_python/python_test.py:221]: Can you compare output0 and output1 against calculated values using input_data_0 and input_data_1 to make sure you get the expected values back.

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

File: qa/L0_backend_python/python_test.py (hunk 3 above)
Comment (Tabrizian) [reply qa/L0_backend_python/python_test.py:221]: I did compare it in the lines below.

**Review by deadeyegoodwin — APPROVED** [review]

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**